doc/tasks/php7cc.md

```diff
@@ -0,0 +1,32 @@
+# Php7cc
+
+The Php7cc task will check PHP 5.3 - 5.6 code compatibility with PHP 7.
+It lives under the `php7cc` namespace and has following configurable parameters:
+
+```yaml
+# grumphp.yml
+parameters:
+    tasks:
+        php7cc:
+            exclude: ['vendor']
+            level: 'error'
```

Comment:
The default level is `~` since the value is null. Can you change this line to use the default value?